tools/tf2openapi/generator/generate.go

```diff
@@ -68,10 +68,10 @@ func (g *Generator) GenerateOpenAPI(model *pb.SavedModel) (string, error) {
 	}
 	json, marshallingErr := spec.MarshalJSON()
 	if marshallingErr != nil {
-		return "", fmt.Errorf(UnmarshallableSpecError, marshallingErr.Error(), json)
+		panic(fmt.Errorf(UnmarshallableSpecError, marshallingErr.Error(), json))
```

Comment:
If you're never erroring, your method signature shouldn't error.

Reply:
Nevermind, just saw above. Seems like we panic when it's unexpected, and don't panic otherwise. Makes sense